### PR TITLE
Events populateInfo

### PR DESCRIPTION
**Pull request by volodymyrvynnyk**

### CODE REVIEW

File: src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java [hunk 1]
```diff
@@ -318,5 +345,11 @@ SELECT COUNT(*)
                 FROM users
                 WHERE account_id = ?
             """;
+
+        public static final String SQL_SELECT_USERNAMES = """
+                SELECT id, first_name, last_name
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java]: You can contact here, please check MySQL CONCAT function

File: src/main/java/com/ita/if103java/ims/dao/WarehouseDao.java [hunk 2]
```diff
@@ -13,6 +14,10 @@
 
     List<Warehouse> findAll();
 
+    Map<Long, String> findWarehouseNames(Long account_id);
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/dao/WarehouseDao.java]: This method is not used, why did you add it?

File: src/main/java/com/ita/if103java/ims/dao/UserDao.java [hunk 3]
```diff
@@ -25,6 +26,10 @@
 
     List<User> findAll(Pageable pageable);
 
+    Map<Long, String> findUsernames(Long accountId);
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/dao/UserDao.java]: This method is not used, why did you add it?

File: src/main/java/com/ita/if103java/ims/dao/WarehouseDao.java (hunk 2 above)
Comment (volodymyrvynnyk) [reply src/main/java/com/ita/if103java/ims/dao/WarehouseDao.java]: In ui, I need to have a list of all workers and warehouses in account for filtering, I'm going to use it
